src/tls.c

```diff
@@ -11616,6 +12074,17 @@ static int TLSX_Write(TLSX* list, byte* output, byte* semaphore,
                 offset += SRTP_WRITE((TlsxSrtp*)extension->data, output+offset);
                 break;
 #endif
+
+#ifdef HAVE_RPK
+            case TLSX_CLIENT_CERTIFICATE_TYPE:
+                offset += CCT_WRITE(extension->data, output + offset, msgType);
```

Comment:
Add WOLFSSL_MSG calls to here and below.

Reply:
Added WOLFSSL_MSG calls.